src/hooks/useUpdateLeave/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { useMutation } from '@tanstack/react-query';
+import { updateLeave } from './request';
+import { Dayjs } from 'dayjs';
+
+const useUpdateLeave = () => {
```

Comment:
merge these hooks to useLeave. getLeave, updateLeave, deleteLeave all should be inside single useLeave hooks